src/content/developers/docs/consensus-mechanisms/pow/mining-algorithms/dagger-hashamoto/index.md

```diff
@@ -0,0 +1,327 @@
+---
+title: Dagger-Hashamoto
+description: A detailed look at the Dagger-Hashamoto algorithm.
+lang: en
+sidebar: true
+incomplete: false
+---
+
+This page includes detailed information about the Dagger Hashamoto mining algorithm. Through a series of substanbtial upgrades, this algorithm was superceded by [Ethash](/ethash). However, this original version has historical significance and is included here because it was an important innovation in Ethereum's development. Proof-of-work mining itself will be deprecated in favour of proof-of-stake during the merge which is expected to happen in summer 2022. To understand the information in this page it is recommended to first read our introductory pages on [proof-of-work consensus](/developers/docs/consensus-mechanisms/pow) and [mining](/developers/docs/consensus-mechanisms/pow/mining).
```

Comment:
```suggestion
Dagger Hashimoto was the original research implementation and specification for Ethereum's mining algorithm. Dagger Hashimoto was superseded by [Ethash](#ethash).
```

Reply:
Looks good, but think we want `/ethash/` as the link, not `#ethash`

```suggestion
Dagger Hashimoto was the original research implementation and specification for Ethereum's mining algorithm. Dagger Hashimoto was superseded by [Ethash](/ethash/).
```